On Solaris, GnuPG's main executable is "gpg2" instead of "gpg".  Perhaps
that should be otherwise, but until we deliver a "gpg" executable, git
should look for "gpg2" instead.  This is not suitable for upstream.

--- git-2.6.1/gpg-interface.c	Fri Oct  2 15:40:47 2015
+++ git-2.6.1/gpg-interface.c	Fri Dec  4 14:12:20 2015
@@ -5,7 +5,7 @@
 #include "sigchain.h"
 
 static char *configured_signing_key;
-static const char *gpg_program = "gpg";
+static const char *gpg_program = "gpg2";
 
 #define PGP_SIGNATURE "-----BEGIN PGP SIGNATURE-----"
 #define PGP_MESSAGE "-----BEGIN PGP MESSAGE-----"
